static/js/components/dashboard/FinancialAidCard.js

```diff
@@ -199,26 +199,37 @@ export default class FinancialAidCard extends React.Component {
 
           <Grid className="financial-aid-box">
             <Cell col={12}>
-                Before you can pay, you need to verify your income. Please mail
-                or fax an English-translated and notarized income tax or income
-                statement document. DO NOT SEND BY EMAIL.
+                Before you can pay, you need to verify your income. Please visit
+                the{" "}
+              <a href="https://na2.docusign.net/Member/PowerFormSigning.aspx?PowerFormId=4a74536d-1629-4709-b8e9-f173a51cf501&env=na2&v=2">
+                  secure website
+              </a>{" "}
+                to upload an English-translated and notarized income tax or
+                income statement document. You can also send documents by mail.
+                DO NOT SEND BY EMAIL.
             </Cell>
             <Cell col={12}>
-              <a onClick={() => setDocsInstructionsVisibility(true)}>
+              <a
+                className="btn-instructions"
+                onClick={() => setDocsInstructionsVisibility(true)}
+              >
                   Read Complete Instructions
               </a>
             </Cell>
           </Grid>
 
           <Grid>
-            <Cell col={1} />
-            <Cell col={5}>Mail to</Cell>
-            <Cell col={6}>or fax</Cell>
+            <Cell col={8}>Upload to DocuSign</Cell>
+            <Cell col={4}>Mail to</Cell>
           </Grid>
 
           <Grid>
-            <Cell col={1} />
-            <Cell col={5}>
+            <Cell col={8}>
+              <a href="https://na2.docusign.net/Member/PowerFormSigning.aspx?PowerFormId=4a74536d-1629-4709-b8e9-f173a51cf501&env=na2&v=2">
+                  https://na2.docusign.net/Member/PowerFormSigning.aspx?PowerFormId=4a74536d-1629-4709-b8e9-f173a51cf501&env=na2&v=2
+              </a>
+            </Cell>
+            <Cell col={4}>
```

Comment:
Can we try changing the number of columns? The URL will have to wrap no matter what. It would be nice if the mailing address didn't have to wrap.